scripts/processIdHere/generateSummary.groovy

```diff
@@ -0,0 +1,17 @@
+package processIdHere
+
+import commons.serviceportal.forms.FormDumper
+import de.seitenbau.serviceportal.scripting.api.v1.ScriptingApiV1
+import de.seitenbau.serviceportal.scripting.api.v1.form.content.FormContentV1
+
+ScriptingApiV1 api = apiV1 // Variable is automatically set by Serviceportal process engine
+FormContentV1 applicantForm = api.getVariable("applicantForm") as FormContentV1
+
+// Generate summary via the FormDumper class
+String summaryAsHtml = ""
+summaryAsHtml += FormDumper.getCssClasses()
```

Comment:
Hier nur ein Hinweis: Anfang Feb Februar wird wird Release 171 auf der OZG-Hub dev installiert, in dem dann auch die CSS-Klassen von der Plattform bereitgestellt werden :) wäre vermutlich eine Gute Idee das auch hier dann gleich einzubinden??

Reply:
Ah, guter Punkt! Ich hab's in SKDE-1961 aufgenommen, dann vergessen wir es nicht, wenn es verfügbar ist, können diesen Branch aber mMn. dann jetzt mergen :)